src/ResourceManager/SiteRecovery/Commands.SiteRecovery/Vault/SetAzureSiteRecoveryVaultSettings.cs

```diff
@@ -29,6 +29,13 @@ public class SetAzureSiteRecoveryVaultSettings : SiteRecoveryCmdletBase
         #region Parameters
 
         /// <summary>
+        /// Gets or sets vault Object.
+        /// </summary>
+        [Parameter(ParameterSetName = ASRParameterSets.Default, Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public System.Object Vault { get; set; }
```

Comment:
Object?  How is this created by the user - how are they going to get this value to pass to the cmdlet?